jinja2cli/cli.py

```diff
@@ -146,6 +150,23 @@ def _load_xml():
     return xmltodict.parse, xml.parsers.expat.ExpatError, MalformedXML
 
 
+def _load_env():
+    def _parse_env(data):
+        """
+        Parse an envfile format of key=value pairs that are newline separated
+        """
+        dict_ = {}
+        for line in data.splitlines():
+            line = line.strip()
```

Comment:
That's not how envfile format works. If you used that value, you'd be getting a key of "foo " and a value of " blah blah blah", and would be intended behavior for environment variables. See my reference material in the issue.